IndexProactiveControls.md

```diff
@@ -0,0 +1,166 @@
+- [1. Define Security Requirements](#1-define-security-requirements)
+- [2. Leverage Security Frameworks and Libraries](#2-leverage-security-frameworks-and-libraries)
+- [3. Secure Database Access](#3-secure-database-access)
+- [4. Encode and Escape Data](#4-encode-and-escape-data)
+- [5. Validate All Inputs](#5-validate-all-inputs)
+- [6. Implement Digital Identity](#6-implement-digital-identity)
+- [7. Enforce Access Controls](#7-enforce-access-controls)
+- [8. Protect Data Everywhere](#8-protect-data-everywhere)
+- [9. Implement Security Logging and Monitoring](#9-implement-security-logging-and-monitoring)
+- [10. Handle All Errors and Exceptions](#10-handle-all-errors-and-exceptions)
+
+# Objective
+
+This index have for objective to indicate to help an [OWASP Proactive Controls](https://www.owasp.org/index.php/OWASP_Proactive_Controls) user to clearly identify which cheat sheets are useful for each section during his usage of the OWASP Proactive Controls.
+
+# Not adressed CS
+
+[AJAX Security Cheat Sheet](cheatsheets/AJAX_Security_Cheat_Sheet.md)
+
+[Cross-Site_Request_Forgery_Prevention_Cheat_Sheet](cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md) - Enforce Access Controls? 
+
+[Docker Security Cheat Sheet](cheatsheets/Docker_Security_Cheat_Sheet.md) -  Leverage Security Frameworks and Libraries?
+
+[Deserialization Cheat Sheet](cheatsheets/Deserialization_Cheat_Sheet.md)
+
+[Denial of Service Cheat Sheet](cheatsheets/Denial_of_Service_Cheat_Sheet.md)
+
+[HTML5 Security Cheat Sheet](cheatsheets/HTML5_Security_Cheat_Sheet.md)
+
+[Mass Assignment Cheat Sheet](cheatsheets/Mass_Assignment_Cheat_Sheet.md)
+
+[PHP Configuration Cheat Sheet](cheatsheets/PHP_Configuration_Cheat_Sheet.md) -  Leverage Security Frameworks and Libraries?
+
+[Protect FileUpload Against Malicious File](cheatsheets/Protect_FileUpload_Against_Malicious_File.md) - Validate All Inputs?
+
+[REST Assessment Cheat Sheet](cheatsheets/REST_Assessment_Cheat_Sheet.md)
+
+[REST Security Cheat Sheet](cheatsheets/REST_Security_Cheat_Sheet.md) - All?
+
+[Ruby on Rails Cheatsheet](cheatsheets/Ruby_on_Rails_Cheatsheet.md) - All?
+
+[Securing Cascading Style Sheets Cheat Sheet](cheatsheets/Securing_Cascading_Style_Sheets_Cheat_Sheet.md)
+
+[Third Party Javascript Management Cheat Sheet](cheatsheets/Third_Party_Javascript_Management_Cheat_Sheet.md)
+
+[User Privacy Protection Cheat Sheet](cheatsheets/User_Privacy_Protection_Cheat_Sheet.md) - Protect Data Everywhere?
+
+[Vulnerability Disclosure Cheat Sheet](cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.md)
+
+[Virtual Patching Cheat Sheet](cheatsheets/Virtual_Patching_Cheat_Sheet.md)
+
+[Web Service Security Cheat Sheet](cheatsheets/Web_Service_Security_Cheat_Sheet.md)
+
+[XML Security Cheat Sheet](cheatsheets/XML_Security_Cheat_Sheet.md)
+
+[XML External Entity Prevention Cheat Sheet](cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.md)
+
+
+
+# 1. Define Security Requirements
```

Comment:
Can you add this [section](https://github.com/OWASP/CheatSheetSeries/blob/master/IndexASVS.md#not-addressed-topic) to the OPC index to indicate that user can create an issue if a CS is mssing for a topic ?

However as here every sections is covered by at least one CS you can omit the text:

*"Section with None content can spot a topic that should be addressed by the Cheat Sheet Series project."*

If possible, can you use the top section order than the ASVS index (it's currently the case but it's a reminder 😃 ):
1. Objective
2. Not adressed topic.
3. OPC sections like currently...